### PR TITLE
Features: Add basic UI

### PR DESCRIPTION
**Pull request by marurevi**
## OOP school library
I created a form of UI for your school library app. This way it can be invoked as an executable and not something you use in IRB exclusively.
- Create the `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain Person).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- In the `main.rb` define the entry point, this will be a method called main that is invoked at the end of your file.

UI

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  end
+
+  def create_person
+    puts 'Enter name:'
+    name = gets.capitalize.chomp
+    puts 'Enter age:'
+    age = gets.chomp.to_i
+    person = Person.new(age, name)
+    @people << person
+  end
```
Comment (VICTOR-CHUKWU) [inline app.rb]: - [ ] your method create person should take two prompt, one for create student and second for create teacher, kindly fix this

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @marurevi  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (marurevi) [reply app.rb]: image

✅ I implemented the required changes

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @marurevi, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Clean/Organized code
:heavy_check_mark: Your app is running as expected. 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

 #### I will leave the comments here, since your `Readme file` is not accessible for this `PR` as you did not make any change on it ####

- [OPTIONAL] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. It's a good thing you are using the template from Microverse but please kindly adapt it to your project. Replace all the information in the various sections of the template with information pertaining to your project.  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**